test/ktn_meta_suite_SUITE.erl

```diff
@@ -14,9 +15,29 @@
 -type config() :: [{atom(), term()}].
 
 -spec init_per_suite(config()) -> config().
+-ifdef(rebar).
+init_per_suite(Config) ->
+    LibDir = code:lib_dir(katana),
+    BuildRoot = filename:dirname(filename:dirname(LibDir)),
+    {ok, Dirlist} = file:list_dir(BuildRoot),
+    [Plt|_] = lists:filter(fun (File) ->
```

Comment:
As a matter of fact, given your code is pretty similar to the one in `ktn_meta_SUITE`, I'm wondering if this whole function clause couldn't just have been replaced by

```erlang
init_per_suite(Config) ->
  LibDir = code:lib_dir(katana),
  [ {elvis_config, filename:join([LibDir, "test", "elvis.config"])}
  , {application, katana}
  | Config
  ].
```